libgalois/src/PropertyViews.cpp

```diff
@@ -19,3 +19,24 @@ katana::internal::ExtractArrays(
 
   return ret;
 }
+
+katana::Result<std::vector<arrow::Array*>>
+katana::internal::ExtractArrays(
+    const PropertyGraph::PropertyView& pview,
+    const std::vector<std::string>& properties) {
+  std::vector<arrow::Array*> ret;
+  for (auto& property : properties) {
+    auto column = pview.GetProperty(property);
+    if (!column) {
+      return ErrorCode::PropertyNotFound;
+    }
+    if (column->num_chunks() != 1) {
+      // Katana form graphs only contain single chunk property columns.
+      return ErrorCode::TODO;
+      // TODO: Maybe we need an InvalidGraph error
```

Comment:
Username? Also add message to error code?

Reply:
Ok.  For the record, I copied this code from the function above.  I guess that author did not provide a username for the TODO or an error message.